app/models/vote.rb

```diff
@@ -0,0 +1,4 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
```

Comment:
You should have a validation to prevent someone from voting for the same work twice.

```suggestion
  belongs_to :work
  
  validates :work_id, uniqueness: { scope: user_id } 
```